webpack.dev.config.js

```diff
@@ -0,0 +1,38 @@
+const { createConfig } = require('@edx/frontend-build');
+
+const path = require('path');
+const fs = require('fs');
+const webpack = require('webpack');
+
+const basePluginsPath = path.resolve(__dirname, 'plugins/');
+
+const pluginTabs = fs.readdirSync(basePluginsPath).map(
+    dir => {
+        console.log(path.resolve(basePluginsPath, dir, 'package.json'));
+        const meta = require(path.resolve(basePluginsPath, dir, 'package.json'));
+        console.log(meta['open-edx-meta'].tabs);
+        return {
+            id: dir,
+            ...meta['open-edx-meta'].tabs,
+        };
+
+    }
+)
+
+pluginTabs.sort((tab1, tab2) => tab1.order > tab2.order);
+
+
+module.exports = createConfig('webpack-dev', {
+    resolve: {
+        alias: {
+            plugins: basePluginsPath,
```

Comment:
Setting this allows us to load plugins as 'plugins/abc' instead of '.../../plugins/abc'. 
It probably doesn't matter in this use case, but I felt it would be a nice way to isolate things.